### ISSUE TITLE
Improve road visibility for the color-blind

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#1939]
In the graphics forum, DaggeTeo wrote: "This may be misplaced but I figure I should mention it anyways... The other day I was showing the game to a friend who suffers slightly from color blindness. He pointed out that the roads were nearly impossible to distinguish from the mountain terrain."

I feel we should explore ways to improve the game experience for the color-blind starting with the roads.

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2012-05-01T13:25:15Z
 * owner: chuckw20
 * the launchpad url was https://bugs.launchpad.net/bugs/992598

**Comment by bunnybot** [comment]
(by hjd)
This is definitely one of those cases where increased accessibility can benefit all players. I am not sure how to improve the current roads, but this should be considered.

**Comment by bunnybot** [comment]
(by fs-launchpad)
I like to add, the colors currently used are so that on some terrain (e.g. the lighter green grassland) the roads are almost invisible even for people with normal ability to distinguish  colors. (This may in a way also be a matter of the used monitor (configuration), but IMHO the colors should be chosen in such a way, that this don't matter).

**Comment by bunnybot** [comment]
(by sirver)
Setting to incomplete for bug sweeping.

**Comment by bunnybot** [comment]
(by janitor)
[Expired for widelands because there has been no activity for 60 days.]

**Comment by bunnybot** [comment]
(by sirver)
I feel this bug is a duplicate, but I cannot find the other bug right now.

**Comment by bunnybot** [comment]
(by chuckw20)
See also Bug #1328637

**Comment by bunnybot** [comment]
(by gunchleoc)
Fixed in build19-rc1.
